sirepo/template/ml.py

```diff
@@ -301,13 +301,26 @@ def stateless_compute_remote_data_bytes_loaded(data):
     return _remote_data_bytes_loaded(data.filename)
 
 
+def stateless_compute_get_archive_file_list(data):
+    return _archive_file_list(data.filename, data.data_type)
+
+
 def write_parameters(data, run_dir, is_parallel):
     pkio.write_text(
         run_dir.join(template_common.PARAMETERS_PYTHON_FILE),
         _generate_parameters_file(data),
     )
 
 
+def _archive_file_list(filename, data_type):
+    def _filter(item):
+        is_dir = getattr(item, a.dir_check)()
+        return is_dir if data_type == "image" else not is_dir
```

Comment:
`is_dir == (data_type =="image")`?

Reply:
I think that passes all files for images and none for text...???